lib/internal/errors.js

```diff
@@ -69,6 +93,7 @@ module.exports = exports = {
   Error: makeNodeError(Error),
   TypeError: makeNodeError(TypeError),
   RangeError: makeNodeError(RangeError),
+  AssertionError: makeAssertionError(),
```

Comment:
I can very well store the constructor reference in a variable named like the class, but that would be a hack!